site/_guides/cert-manager.md

```diff
@@ -164,10 +163,13 @@ metadata:
 spec:
   acme:
     email: user@example.com
-    http01: {}
     privateKeySecretRef:
-      name: letsencrypt-staging
+      name: letsencrypt-prod
```

Comment:
Is this change intention? The context of this file is the staging let's encrypt server

Reply:
Nope, will revert.